addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -114,16 +114,27 @@ publicWidget.registry.popup = PopupWidget;
 // Prevent bootstrap to prevent scrolling and to add the strange body
 // padding-right they add if the popup does not use a backdrop (especially
 // important for default cookie bar).
```

Comment:
Maybe update the comment to match your use case...?

Reply:
Comment is kinda up-to-date here, I would however add a comment in the scss next to the `s_popup_overflow_page` to explain the case.